.gitignore

```diff
@@ -117,7 +117,6 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.toptal.com/developers/gitignore/api/linux,macos,windows,particle,c++
-docs/
```

Comment:
Note that I am specifically ignoring docs for doxygen.  We can probably move doxygen output somewhere else.

Reply:
Should I put the docs in a differently named folder then?

Reply:
no